homeassistant/components/tplink_omada/switch.py

```diff
@@ -96,109 +176,82 @@ def _wan_connect_disconnect(
     )
 
 
-GATEWAY_PORT_SWITCHES: list[GatewayPortSwitchEntityDescription] = [
-    GatewayPortSwitchEntityDescription(
+SWITCH_PORT_DETAILS_SWITCHES: list[OmadaSwitchPortSwitchEntityDescription] = [
+    OmadaSwitchPortSwitchEntityDescription(
+        key="poe",
+        translation_key="poe_control",
+        exists_func=lambda d, p: d.device_capabilities.supports_poe
```

Comment:
Please put parentheses around the lambda and indent it so it's easier to separate it from the entity description parameters.